config/resources/government.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "kans": "http://kanselarij.vo.data.gift/core/",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "ext": "http://mu.semte.ch/vocabularies/ext/"
+  },
+  "resources": {
+    "government-fields": {
+      "new-resource-base": "http://themis.vlaanderen.be/beleidsveld/",
+      "class": "kans:Beleidsveld",
+      "attributes": {
+        "prefLabel": {
```

Comment:
We gebruiken kebab case om de attributen en relaties te specifieren, dus `pref-label` ipv `prefLabel`. De Ember frontend zal automatisch de vertaalslag maken van kebab naar camel case. Deze opmerking komt nog op een aantal plaatsen voor in deze file.